virttest/utils_misc.py

```diff
@@ -851,6 +851,44 @@ def archive_as_tarball(source_dir, dest_dir, tarball_name=None,
     tarball.close()
 
 
+def get_guest_service_status(session, service, service_former=None):
+    """
+    Get service's status in guest.
+    Return 'active' for 'running' and 'active' case.
+    Return 'inactive' for 'stopped' and 'inactive' case.
+
+    :param session: An Expect or ShellSession instance to operate on
+    :type devices: Expect or ShellSession class
+    :param service: Service name that we want to check status.
+    :type service: String
+    :param service_former: Service's former name
+    :type service: String
+    :return: 'active' or 'inactive'
+    :rtype: String
+    """
+    output = ""
+    try:
+        session.cmd("systemctl --version")
+        cmd = "systemctl status %s.service" % service
+        output = session.cmd_output(cmd)
+    except Exception:
+        if service_former is not None:
```

Comment:
In general, you use `is not None` when you are specifically testing for `None`, and absolutely does not want an expression that evaluates to a boolean `False`. Otherwise `is not [variable]` is the preferred way of doing things.